lib/xeroizer/models/credit_note.rb

```diff
@@ -126,8 +126,30 @@ def total(always_summary = false)
         def pdf(filename = nil)
           parent.pdf(id, filename)
         end
-              
+
+        def save
+          # Calling parse_save_response() on the credit note will wipe out
+          # the allocations, so we have to manually preserve them.
+          allocations_backup = allocations
+          super
+          allocations = allocations_backup
+          allocate if !allocations.empty?
+        end
```

Comment:
This should always return `true` to match `Xeroizer::Record::Base#save` behaviour.
Specially for people using it like this:
```
if credit_note.save then
  do_something
end
```